test/YesSql.Tests/CoreTests.cs

```diff
@@ -218,14 +219,20 @@ public void CreateTables(IConfiguration configuration)
 
                     builder.CreateMapIndexTable<PersonByName>(column => column
                             .Column<string>(nameof(PersonByName.SomeName)),
-                            "Collection1"
+                            "Col1"
                             );
 
                     builder.CreateMapIndexTable<PersonByNameCol>(column => column
                             .Column<string>(nameof(PersonByNameCol.Name)),
-                            "Collection1"
+                            "Col1"
                             );
 
+                   builder.CreateReduceIndexTable<PersonsByNameCol>(column => column
```

Comment:
How does it work? Being able to use the same table for two types of indexes. Or are the names of reduce index generated differently?

Reply:
By the `Col1` down the bottom. More explicitly it's `collection: Col1` so the table eventually becomes `Col1_PersonsByNameCol`

Reply:
Actually I didn't notice the "s" in Persons, because there is also a PersonByName index